internal/app/backend/backend_service.go

```diff
@@ -61,6 +63,21 @@ var (
 	})
 )
 
+func newBackendService(cfg config.View) (*backendService, error) {
+	conn, err := rpc.GRPCClientFromConfig(cfg, "api.synchronizer")
+	if err != nil {
+		logger.Errorf("Failed to get Synchronizer connection, %v.", err)
```

Comment:
If you're logging an error before returning it then don't log it.

Reply:
Removed.